single-pool/js/packages/client/src/quarantine.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  getBase58EncodedAddressCodec,
+  Base58EncodedAddress,
+  AccountRole,
+  getProgramDerivedAddress,
+} from '@solana/web3.js';
+
+// HERE BE DRAGONS
+// this is all the stuff that shouldnt be in our library once we can import from elsewhere
+
+function address<TAddress extends string>(string: TAddress): Base58EncodedAddress<TAddress> {
+  return string as Base58EncodedAddress<TAddress>;
+}
+
+export const SYSTEM_PROGRAM_ID = address('11111111111111111111111111111111');
+export const STAKE_PROGRAM_ID = address('Stake11111111111111111111111111111111111111');
+export const SYSVAR_RENT_ID = address('SysvarRent111111111111111111111111111111111');
+export const SYSVAR_CLOCK_ID = address('SysvarC1ock11111111111111111111111111111111');
+export const SYSVAR_STAKE_HISTORY_ID = address('SysvarStakeHistory1111111111111111111111111');
+export const STAKE_CONFIG_ID = address('StakeConfig11111111111111111111111111111111');
+export const STAKE_ACCOUNT_SIZE = 200n;
+
+export const TOKEN_PROGRAM_ID = address('TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA');
+export const ATOKEN_PROGRAM_ID = address('ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL');
```

Comment:
I dunno; I think program clients should own their own config like this. What if a program upgraded, redeployed somewhere else, and our hardcoded program ID fell out of sync with reality?

Reply:
I would think that's extremely unlikely for each of these

Reply:
it makes sense from a hygiene perspective for clients to define their own ids but im not sure what the current status is on there being official clients (if we use umi can we just build and release the built clients as official?)

Reply:
The leading idea is whoever owns the program should be responsible for generating and publishing a corresponding client, so yeah we'd have to gen then publish clients for each program we own, basically this whole list lol

Reply:
But I'm not a huge fan of adding a package as a dependency for just a constant. Sure, if you add it to dev dependencies it will shake away, but will everyone do that?

Reply:
fwiw i remain of the opinion that web3 should ship with full system/stake/vote clients